dotnet/src/webdriver/Internal/Logging/ConsoleLogHandler.cs

```diff
@@ -0,0 +1,56 @@
+// <copyright file="ConsoleLogHandler.cs" company="WebDriver Committers">
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements. See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership. The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+
+namespace OpenQA.Selenium.Internal.Logging
+{
+    /// <summary>
+    /// Represents a log handler that writes log events to the console.
+    /// </summary>
+    public class ConsoleLogHandler : ILogHandler
+    {
+        private static readonly Dictionary<LogEventLevel, string> _levelMap = new Dictionary<LogEventLevel, string>
+        {
+            { LogEventLevel.Trace, "TRC" },
+            { LogEventLevel.Debug, "DBG" },
+            { LogEventLevel.Info, "INF" },
+            { LogEventLevel.Warn, "WRN" },
+            { LogEventLevel.Error, "ERR" }
```

Comment:
This is subjective. But anyway, I would prefer log levels this way.

```suggestion
            { LogEventLevel.Trace, "TRACE" },
            { LogEventLevel.Debug, "DEBUG" },
            { LogEventLevel.Info, " INFO" },
            { LogEventLevel.Warn, " WARN" },
            { LogEventLevel.Error, "ERROR" }
```

Reply:
Idea here is to make output prefix beeing length-fixed:
```
16:36:10.763 TRC SeleniumManager: Driver path: C:\Users\nikol\.cache\selenium\chromedriver\win64\119.0.6045.105\chromedriver.exe
16:36:10.836 DBG HttpCommandExecutor: Executing command: []: newSession {"capabilities":{"firstMatch":[{"browserName":"chrome","
```

Reply:
`" INFO"` and `" WARN"`, which are shorter than 5 characters, can be prefixed with a space character.

Reply:
Also works. both are lenght-fixed. @titusfortner what do you prefer?

Reply:
Yeah, harder to read the abbreviations I think

Reply:
From the other hand why not `Information` rather than `Info`, or `Warning` rather that `Warn`.

Reply:
Java & Python do `Warning`, Ruby does `Warn`, so either is fine. I'm not worried about the spacing, either. Most of this is convention for the given language, but Info, Warn, Debug are all English words.

Reply:
Did it like @YevgeniyShunevych proposed.

```
"TRACE", "DEBUG", " INFO", " WARN", "ERROR"
```